doc/related_projects.rst

```diff
@@ -55,6 +55,11 @@ enhance the functionality of scikit-learn's estimators.
   wrapper around scikit-learn that makes it easy to run machine learning
   experiments with multiple learners and large feature sets.
 
+- `Xcessiv <https://github.com/reiinakano/xcessiv>`_ is a notebook-like
+  application for quick, scalable, and automated hyperparameter tuning
+  and stacked ensembling. Painlessly keep track of hundreds of model-hyperparameter
```

Comment:
I'd rather avoid hyperbolic marketing speak. Can we leave it at the first sentence, or at least remove "Painlessly" from the second?